.github/workflows/issue-comment.yml

```
@@ -0,0 +1,27 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment to new issue
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+          body: `👋 Thanks for opening this issue and contributing!
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 22: found character that cannot start any token

(syntax-check)